src/vic-stdlib.c

```diff
@@ -48,28 +48,19 @@ void vic_get(void)
 
 void vic_list(void)
 {
-    char option = '\0';
-    char *args_format = "-%c ";
-
-    if (vic_args(args_format, &option) != 1) {
-        vic_print_err(VIC_ERR_INVALID_ARGS);
-        return;
+    uint8_t i;
+    for (i = 0; i < vic_funcs_len; i++) {
+        if (vic_funcs[i].p_func != NULL) {
+            vic_println(vic_funcs[i].name);
+        }
     }
+}
 
+void vic_list_vars(void)
+{
     uint8_t i;
-    if (option == 'f') {
-        for (i = 0; i < vic_funcs_len; i++) {
-            if (vic_funcs[i].p_func != NULL) {
-                vic_println(vic_funcs[i].name);
-            }
-        }
-    } else if (option == 'v') {
-        for (i = 0; i < vic_vars_len; i++) {
-            vic_println(vic_vars[i].name);
-        }
-    } else {
-        vic_print_err(VIC_ERR_INVALID_ARGS);
-        return;
+    for (i = 0; i < vic_vars_len; i++) {
+        vic_println(vic_vars[i].name);
```

Comment:
@vargac the old vic used to also print the address of the function.

I do not think that is necessary anymore.